src/blueteam/LumberJack.java

```diff
@@ -32,54 +33,127 @@
 		moveDirection = randomDirection();
 	}
 
+	boolean friendlyStrikeDamage() {
+		return rc.senseNearbyRobots(GameConstants.LUMBERJACK_STRIKE_RADIUS, rc.getTeam()).length == 0 ? false : true;
```

Comment:
It is correct. When you don't have any friendly units around you you want to return FALSE -> meaning that there will be no friendly damage.